core/src/main/java/io/paradoxical/cassieq/resources/api/v1/QueueResource.java

```diff
@@ -145,7 +165,7 @@ public Response putMessage(
             String message) {
 
         final Optional<QueueDefinition> queueDefinition = getQueueDefinition(queueName);
```

Comment:
maybe `getQueueDefinition` should just filter the optional. rename it to `getActiveQueueDefinition`